pkg/sfu/buffer/rtpstats.go

```diff
@@ -25,6 +24,28 @@ func getPos(sn uint16) (uint16, uint16) {
 	return sn >> 6, sn & 0x3f
 }
 
+type RTPFlowState struct {
+	IsHighestSN        bool
+	HasLoss            bool
+	LossStartInclusive uint16
+	LossEndExclusive   uint16
+}
+
+type RTPSnapshotInfo struct {
+	PacketsExpected uint32
+	PacketsLost     uint32
+	MaxJitter       float64
+	MaxRtt          uint32
+}
+
+type Snapshot struct {
```

Comment:
This is the snap shot. It is very light weight now. It does
- sets the start sequence number when a snapshot is updated (i. e. when moving from one window to next)
- updates quantities for which we maintain max (as of now it is RTT and Jitter only)

In the future if this snap shot becomes heavier, need to watch out for unnecessary use. Hope is that this should be kept lightweight.